capstone/src/main/java/com/google/sps/servlets/clubs/OfficerServlet.java

```diff
@@ -0,0 +1,52 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/* Servlet that stores and returns data relating to clubs. */
+@WebServlet("/officer")
+public class OfficerServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String club = request.getParameter(Constants.CLUB_NAME_PROP);
+    UserService userService = UserServiceFactory.getUserService();
+    String userEmail = userService.getCurrentUser().getEmail();
+    boolean isOfficer = isUserOfficer(userEmail, club);
+    response.setContentType("text/plain");
+    response.getWriter().println(isOfficer);
+  }
+
+  public boolean isUserOfficer(String user, String clubName) {
```

Comment:
Does this method need to be public? If it's for testing, and there's no way to test if this is private, then could you change this to package-scoped (remove the public keyword; should work since the tests are defined in the same package) and add a comment that this is package-scoped for testing?

Reply:
I wanted to somehow make this method accessible to other code so people don't need to rewrite this functionality - would a sensible way to do this be to make it package-scoped and static?